lint/backend.py

```diff
@@ -109,7 +109,9 @@ def reduced_concurrency():
         yield
 
 
-def finalize_errors(view, errors, offset):
+def finalize_errors(linter, errors, offset):
+    linter_name = linter.name
```

Comment:
Any particular reason for the intermediate var?

Reply:
That's just automatic writing style bc after that is a for loop. And you just put everything that cannot change out of it.

It's not for performance, but to keep the changing parts from the constants.